backend/src/driver.js

```diff
@@ -0,0 +1,8 @@
+import neo4j from 'neo4j-driver';
+import {NEO4J_HOST, NEO4J_PASSWORD, NEO4J_USERNAME} from "./config";
+
+const driver = neo4j.driver(
```

Comment:
⭐ ⭐ Data created in mutations are persisted and your queries and mutations still work as expected.